blog/2020-12-16-cwa-release-1-9/index.md

```diff
@@ -18,7 +18,7 @@ If a user had multiple encounters with a high risk (red), he or she will see the
 
 <br></br>
 
-<center> <img src="./cwa-high-and-low-risk.jpg" title="Corona-Warn-App High and Low Risk" alt="Corona-Warn-App High and Low Risk" style="align: center"> </center>
+<div style="text-align:center"> <img src="./cwa-high-and-low-risk.jpg" title="Corona-Warn-App High and Low Risk" alt="Corona-Warn-App High and Low Risk" style="align: center"> <div>

```

Comment:
The end tag `</center>` has been converted to the start tag `<div>`. This is wrong and causes Google chrome to then center the following text.

Reply:
Seems to be resolved